### ISSUE TITLE
DirectX

### ISSUE THREAD

**Issue by vavavr00m** [jbara2002/redist-pack#2]
1. Are all DirectX versions included? Some older games may need specific DirectX versions
2. I couldn't find any DirectX version installed under letter D or Microsoft

![image](https://github.com/jbara2002/redist-pack/assets/4675132/c57a4bc8-1cc0-4f7c-a441-3c05b1e8ad24)

![image](https://github.com/jbara2002/redist-pack/assets/4675132/1711c8d8-1f2f-4258-b9b7-009bc0f12a19)

**Comment by jbara2002** [comment]
yes installs DirectX 9.0c + UWP version of DirectX. Also enables DirectPlay Feature from settings.😊
